offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/AppView.groovy

```diff
@@ -167,10 +167,10 @@ class AppView extends VerticalLayout {
 
         TomatoFeatures() {
             this.createOfferBtn = new Button("New Offer")
-            this.createCustomerBtn = new Button("New Customer")
+            this.createCustomerBtn = new Button("New Person")
```

Comment:
I'd suggest keeping the variable name similiar to the actual label (You have to change it in the rest of the code as well of course) 👍 
```suggestion
            this.createPersonBtn = new Button("New Person")
```

Reply:
done 4c6e56b